schema/me/update_collector_profile.js

```diff
@@ -1,14 +1,17 @@
 import gravity from '../../lib/loaders/gravity';
-import { CollectorProfileType } from './collector_profile';
+import { CollectorProfileFields } from './collector_profile';
 import {
   GraphQLBoolean,
   GraphQLString,
 } from 'graphql';
+import { mutationWithClientMutationId } from 'graphql-relay';
 
-export default {
-  type: CollectorProfileType,
+let UpdateCollectorProfile;
```

Comment:
actually i dont even need to store it anywhere, i can just do ``export default mutationWith...`